manticore/manticore.py

```diff
@@ -224,9 +224,12 @@ def unregister_plugin(self, plugin):
         plugin.manticore = None
 
     def __del__(self):
-        plugins = list(self.plugins)
-        for plugin in plugins:
-            self.unregister_plugin(plugin)
+        try:
+            plugins = list(self.plugins)
+            for plugin in plugins:
+                self.unregister_plugin(plugin)
+        except:
+            pass
```

Comment:
What are the cases when it fails? Maybe we should wrap only the `self.unregister_plugin` call with `try ... except`?